app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+from __future__ import annotations
+
+
+class SoftwareEngineer:
+    skills: list = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills: list = self.skills + ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills: list = self.skills + ["Python", "SQL", "Django"]
```

Comment:
use `+=`

Reply:
for some reason I added the annotation of types, but it did not allow me to write += I used a simple way)